src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4120,6 +4120,45 @@ class spell_gen_mixology_bonus : public SpellScriptLoader
     }
 };
 
+enum LandmineKnockbackAchievement
+{
+    SPELL_LANDMINE_KNOCKBACK_ACHIEVEMENT = 57064
+};
+
+class spell_gen_landmine_knockback_achievement : public SpellScriptLoader
+{
+    public:
+        spell_gen_landmine_knockback_achievement() : SpellScriptLoader("spell_gen_landmine_knockback_achievement") { }
+
+        class spell_gen_landmine_knockback_achievement_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_gen_landmine_knockback_achievement_SpellScript);
+
+            void HandleScript(SpellEffIndex /*effIndex*/)
+            {
+                if (Player* target = GetHitPlayer())
+                {
+                    Aura const* aura = GetHitAura();
+                    if (!aura || aura->GetStackAmount() < 10)
+                        return;
+
+                    target->CastSpell(target, SPELL_LANDMINE_KNOCKBACK_ACHIEVEMENT, true);
+                }
+            }
+
+            void Register() override
+            {
+                OnEffectHitTarget += SpellEffectFn(spell_gen_landmine_knockback_achievement_SpellScript::HandleScript, EFFECT_0, SPELL_EFFECT_SCRIPT_EFFECT);
+                OnEffectHitTarget += SpellEffectFn(spell_gen_landmine_knockback_achievement_SpellScript::HandleScript, EFFECT_1, SPELL_EFFECT_APPLY_AURA);
```

Comment:
OK, will remove it. It was more or less a test if it would help in troubleshooting.